CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
-## master / unreleased
+## 0.13.0 / 2023-06-21
```

Comment:
Not feedback on this specific PR, but something to think about in general: I think we could stop posting the dates here. The github release page will have the release date and not having dates gives us the flexibility for reviews to take a few days if need be.

Reply:
The date thing is a promu requirement. I guess we would have to discuss that with the rest of prometheus-dev.